src/routes.tsx

```diff
@@ -82,7 +82,7 @@ export class RootComponent extends React.Component<RootComponentProps, {}> {
             onEnter={this.requireAuth.bind(this)} />
         <IndexRoute
             component={Controls} />
-        <IndexRedirect to="controls" />
+        <Redirect path="*" to="controls" />
```

Comment:
I'm going to merge this one because it's better than not having a 404 page, but can you toss in a "NOT FOUND" route. Just some simple "Could not find that" verbiage so people understand what's going on.